src/server/db/story-dbConfiguration.js

```diff
@@ -170,3 +191,16 @@ sjrk.storyTelling.server.storiesDb.validateFunction = function (newDoc) {
         throw ({forbidden: "doc.type is required"});
     }
 };
+
+/**
+ * This function is used as a "view" design doc once it's migrated to CouchDB.
+ * Will locate user/author accounts.
+ *
+ * @param {Object} doc - a document to evaluate in the view
+ */
+sjrk.storyTelling.server.storiesDb.byUsernameOrEmail = function (doc) {
+    if (doc.type === "user") {
+        emit(doc.username, doc);
```

Comment:
I know nothing about the client side logic. As the server logic in this pull request always returns the email address as a login success signal, I wonder if a user could login via either the username or email address, or just email address. In the case of the latter, it might be better to remove this line in order to reduce the index volume and prevent a potential security issue.

Reply:
There are no explicit usernames anymore, only an e-mail address to identify authors. Thanks for spotting this leftover bit. I've removed it, and have also renamed the name of the function. There are some other references to byUsernameOrEmail but these are used internally by fluid-express-user, so I have left them as is.